src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java

```diff
@@ -106,60 +110,71 @@ public void consume(Message<String> message) {
             String businessProcess = (String) mh.get(BUSINESS_PROCESS_HEADER_KEY);
 
             if (checkTso(tsos, tso, format, businessProcess)) {
```

Comment:
Maybe it is not the best way to implement it, I would have put the process config loop here.